app/src/main/java/ch/epfl/cs311/wanderwave/model/data/Profile.kt

```diff
@@ -0,0 +1,12 @@
+package ch.epfl.cs311.wanderwave.model.data
+
+import android.net.Uri
+
+data class Profile(
+    var firstName: String,
+    var lastName: String,
+    var description: String,
+    var numberOfLikes: Int,
+    var isPublic: Boolean,
+    var profilePictureUri: Uri? = null
+)
```

Comment:
Should we include a field for a currently "on repeat" song. It could be like a user status. We can maybe restrict this feature so that people should first collect points to unlock?

Reply:
Speaking of points, I think we can also add the field for the currency.